examples/queues/failed.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021, Nitric Technologies Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [START import]
+import { queues } from '@nitric/sdk';
+// [END import]
+
+export async function queueFailed() {
+  // [START snippet]
+  // Publish a collection of tasks
+  const failedMessages = await queues()
+    .queue('my-queue')
+    .send({ id: '1234'})
+
+  // Check that it returned Failed Messages
+  if (failedMessages instanceof Object) {
+    for (const message in failedMessages) {
+      console.log(message)
+    }
+  }
+  // [END snippet]
+}
```

Comment:
We should fix the implementation to avoid the weird `void | FailedTasks[]` typing here. We should remove the use of the `send` method in the send implementation and remove `sendBatch` (only have send using the gRPC sendbatch method.

Still accept singular or array but always convert singular to an array. Then always return an array of failed tasks.

Reply:
Pull request in to fix that issue:
https://github.com/nitrictech/node-sdk/pull/68